doc/metrics.md

```diff
@@ -0,0 +1,108 @@
+Using metrics
+=============
+
+## Available metrics
+
+* `go_*` - a set of defult Go metrics provided by Prometheus library
+* `process_*` - a set of defult Process metrics provided by Prometheus library
+* `ncagent_report_count_total` (label `agent`) - Counter. Number of total
+   reports from every agent (agents separated by label).
+* `ncagent_error_count_total` (label `agent`) - Counter. Number of total errors
+   from every agent (agents separated by label). This counter is incremented
+   when agent does not report within `reporting_interval * 2` timeframe.
+
```

Comment:
Those are different agents from metrics point of view, we described hostnet/private in the main README doc.